2021/27xxx/CVE-2021-27773.json

```diff
@@ -1,18 +1,92 @@
 {
```

Comment:
'value' is a required property

Reply:
Forgot to delete the extra field which was not needed.